src/OpenTelemetry.AutoInstrumentation/Instrumentations/Logger/LoggingBuilderIntegration.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="LoggingBuilderIntegration.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.AutoInstrumentation.CallTarget;
+
+namespace OpenTelemetry.AutoInstrumentation.Instrumentations.Logger;
+
+/// <summary>
+/// Microsoft.Extensions.Logging.LoggingBuilder calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Microsoft.Extensions.Logging",
+    TypeName = "Microsoft.Extensions.Logging.LoggingBuilder",
+    MethodName = ".ctor",
+    ReturnTypeName = ClrNames.Void,
```

Comment:
Returned type is tricky for me. I suppose that it should be `LoggingBuilder` as it is a constructor.
As I remember the returned type is nowhere verified while rejiting/building bytecode.

Same should be applied for `instrumentation.json`

Reply:
I had the similar thought, but constructor has a return type as void in both our repo and in DataDog.

Reference: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/2072410667c15e1804fe71e36d4501bca3057082/src/OpenTelemetry.AutoInstrumentation/Instrumentations/MongoDb/MongoClientIntegration.cs#L35

Reply:
Remember with have access to the instance inside the constructor. The instance is built with `newobj` and then that instance to call the actual constructor.